env/file_system_tracer.h

```diff
@@ -133,10 +133,10 @@ class FileSystemPtr {
 // overridden.
 class FSSequentialFileTracingWrapper : public FSSequentialFileWrapper {
  public:
-  FSSequentialFileTracingWrapper(FSSequentialFile* t,
+  FSSequentialFileTracingWrapper(std::unique_ptr<FSSequentialFile>&& t,
```

Comment:
I personally don't like && for unique_ptr parameters (unnecessary apparent complexity), and neither does Google C++ style. But our codebase currently uses unique_ptr&& in many places and there doesn't seem to be a C++ community consensus. http://scottmeyers.blogspot.com/2014/07/should-move-only-types-ever-be-passed.html